modules/ding_seo/ding_seo.features.metatag.inc

```diff
@@ -0,0 +1,245 @@
+      'schema_organization.image' => array(
+        'value' => 'a:4:{s:5:"@type";s:11:"ImageObject";s:3:"url";s:61:"[node:field_ding_library_title_image:ding_primary_large:path]";s:5:"width";s:62:"[node:field_ding_library_title_image:ding_primary_large:width]";s:6:"height";s:63:"[node:field_ding_library_title_image:ding_primary_large:height]";}',
+      ),
+      'schema_organization.logo' => array(
+        'value' => 'a:4:{s:5:"@type";s:11:"ImageObject";s:3:"url";s:26:"[ding_seo_logo_tokens:url]";s:5:"width";s:28:"[ding_seo_logo_tokens:width]";s:6:"height";s:29:"[ding_seo_logo_tokens:height]";}',
+      ),
+    ),
+  );
+
+  // Exported Metatag config instance: node:ding_news.
+  $config['node:ding_news'] = array(
+    'instance' => 'node:ding_news',
+    'disabled' => FALSE,
+    'config' => array(
+      'og:url' => array(
+        'value' => '[node:url:absolute]',
+      ),
+      'og:title' => array(
+        'value' => 'node:title]',
+      ),
+      'og:description' => array(
+        'value' => '[node:field_ding_news_lead]',
+      ),
+      'og:updated_time' => array(
+        'value' => '[node:changed:custom:c]',
+      ),
+      'og:image' => array(
+        'value' => '[node:field_ding_news_title_image:ding_secondary_large:path]',
+      ),
+      'og:image:type' => array(
+        'value' => '[node:field_ding_news_title_image:ding_secondary_large:mimetype]',
+      ),
+      'og:image:width' => array(
+        'value' => '[node:field_ding_news_title_image:ding_secondary_large:width]',
+      ),
+      'og:image:height' => array(
+        'value' => '[node:field_ding_news_title_image:ding_secondary_large:height]',
+      ),
+      'article:published_time' => array(
+        'value' => '[node:created:custom:c]',
+      ),
+      'article:modified_time' => array(
+        'value' => '[node:changed:custom:c]',
+      ),
+      'schema_article.@type' => array(
+        'value' => 'NewsArticle',
+      ),
+      'schema_article.headline' => array(
+        'value' => '[node:title]',
+      ),
+      'schema_article.name' => array(
+        'value' => '[node:title]',
+      ),
+      'schema_article.description' => array(
+        'value' => '[node:field_ding_news_lead]',
+      ),
+      'schema_article.author' => array(
+        'value' => 'a:5:{s:5:"@type";s:12:"Organization";s:3:"@id";s:10:"[site:url]";s:4:"name";s:11:"[site:name]";s:3:"url";s:10:"[site:url]";s:4:"logo";a:4:{s:5:"@type";s:11:"ImageObject";s:3:"url";s:26:"[ding_seo_logo_tokens:url]";s:5:"width";s:28:"[ding_seo_logo_tokens:width]";s:6:"height";s:29:"[ding_seo_logo_tokens:height]";}}',
+      ),
+      'schema_article.publisher' => array(
+        'value' => 'a:5:{s:5:"@type";s:12:"Organization";s:3:"@id";s:10:"[site:url]";s:4:"name";s:11:"[site:name]";s:3:"url";s:10:"[site:url]";s:4:"logo";a:4:{s:5:"@type";s:11:"ImageObject";s:3:"url";s:26:"[ding_seo_logo_tokens:url]";s:5:"width";s:28:"[ding_seo_logo_tokens:width]";s:6:"height";s:29:"[ding_seo_logo_tokens:height]";}}',
```

Comment:
I actually don't know why it was done this way. Perhaps something with anonymity. It doesn't really make sense though, since the auther is shown on the page anyway. Changed it to use the node author instead.